test/acceptance/integration/Rules/EPL/epl_create_rules.feature

```diff
@@ -71,6 +71,21 @@ Feature: Append a new rule in Perseo manager
     Then I receive an "200" http code in rules request
     And Validate that rule name is created successfully in db
 
+  @happy_path
+  Scenario: Append a sms multiple rule in Perseo manager
```

Comment:
Apart from this acceptance test (which I'm not sure is in use now), I'd suggest adding a component test. You can base in existing "should return ok with a valid action with a rule for sms", just adding a new case (a new JSON) which the new syntaxis for multi-SMS.